ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/tracing/diff/StateDiffGenerator.java

```diff
@@ -38,7 +38,7 @@ public class StateDiffGenerator {
   public Stream<Trace> generateStateDiff(final TransactionTrace transactionTrace) {
     final List<TraceFrame> traceFrames = transactionTrace.getTraceFrames();
     if (traceFrames.isEmpty()) {
-      return Stream.empty();
+      throw new RuntimeException("expected to have at least one processed frame");
```

Comment:
This change in behavior is not essential to the precompile trace fix.  Please consider moving it to one of the other RPC PRs.